main.rb

```diff
@@ -0,0 +1,63 @@
+class Main
+  def initialize
+    @choice = 0
+    @books = []
+    @music_albums = []
+    @games = []
+    @genres = []
+    @authors = []
+    @labels = []
+  end
+
+  def run_app
+    put 'Welcome to the catalog!'
+    sleep 0.75
+  end
+
```

Comment:
Dear @OLIPLICHE,

I think you wanted to write `puts` instead of `put` in line 13.

Reply:
Done it. thank you